tests/e2e/specs/in-game/layout.spec.js

```diff
@@ -219,18 +214,17 @@ describe('Four dialogs layout', () => {
   });
 });
 
-describe.skip('Aesthetic tests', () => {
+describe('Aesthetic tests', () => {
```

Comment:
Maybe we do want to skip these 
```suggestion
describe.skip('Aesthetic tests', () => {
```